templates/edge-computing/index.html

```diff
@@ -10,468 +13,457 @@
-          {{ image(url="https://assets.ubuntu.com/v1/dfb83f4e-Edge-telecoms.svg",
-                    alt="",
-                    width="211",
-                    height="140",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
-        </div>
-        <h2>Telco edge: 5G & edge computing</h2>
-        <ul class="p-list">
-          <li class="p-list__item is-ticked">Lightweight to run on small servers constrained by real estate</li>
-          <li class="p-list__item is-ticked">API-driven for providing on-demand compute and services</li>
-          <li class="p-list__item is-ticked">Distributed workloads, serving compute near the consumer</li>
-        </ul>
+      <div class="col">
         <p>
-          <a href="/telco">Read more on Telco@Edge&nbsp;&rsaquo;</a>
+          The Canonical team can deploy the first site with you to enable your edge strategy and upskill your teams. We provide long-term security maintenance and technical support for all the open source components you need to build your micro cloud.
         </p>
+        <hr class="p-rule--muted" />
+        <a href="/contact-us" class="js-invoke-modal p-button">Get in touch</a>
       </div>
     </div>
   </section>
 
-  <section class="p-strip is-bordered">
-    <div class="row">
-      <div class="col-6">
-        <div class="u-sv3 u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/b6a48d38-Edge-business.svg",
-                    alt="",
-                    width="232",
-                    height="110",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <div class="col">
+          <h2>
+            Integrate with
+            <br />
+            the best hardware vendors
+          </h2>
         </div>
-        <h2>The future of remote-work with edge computing</h2>
-        <ul class="p-list">
-          <li class="p-list__item is-ticked">Low-touch, self-healing, few to no human interactions</li>
-          <li class="p-list__item is-ticked">Central remote management for corporate policies</li>
-          <li class="p-list__item is-ticked">On-demand K8s clusters, virtual machines, and containers</li>
-        </ul>
-        <p>
-          <a href="/desktop/organisations">Read about Ubuntu in organisations&nbsp;&rsaquo;</a>
-        </p>
-      </div>
-      <div class="col-6">
-        <div class="u-sv3 u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/5e4aba82-Edge-IoT.svg",
-                    alt="",
-                    width="314",
-                    height="110",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
+        <div class="col">
+          <p>Ubuntu works with a number of hardware vendors specializsed in edge computing server equipment.</p>
```

Comment:
Spelling for "specializsed"
```suggestion
          <p>Ubuntu works with a number of hardware vendors specialized in edge computing server equipment.</p>
```